test/cucumber/support/viewGraphics.feature

```diff
@@ -11,25 +11,21 @@ Feature: Visualização de gráficos de coletas
     And Nenhuma coleta foi realizada na última semana
     When Eu seleciono a opção “Última Semana”
     Then Eu posso ver uma tela com um gráfico vazio
-    And O intervalo do eixo x é [7/10/2016, 13/10/2016]
-
 
   Scenario: Seleção de gráfico de coletas da última semana
     Given Eu estou na tela de opções de gráficos
     And Foram realizadas apenas 3 coletas no dia “16/10/2016” e apenas 2 coletas no dia “13/10/2016”
     And A data atual é 17/10/2016
     When Eu seleciono a opção “Última semana”
-    Then Eu posso ver um gráfico Coletas x Data  cuja altura é 2 quando x = “13/10/2016” e 3 quando x = “16/10/2016”
-    And O intervalo do eixo x é [11/10/2016, 17/10/2016]
+    Then Eu posso ver um gráfico Coletas x Data que possui apenas as alturas 2 quando x = “13/10/2016” e 3 quando x = “16/10/2016”
```

Comment:
Alguns then tem textos muito grandes, que poderiam ser divididos menores usando "and"